plan/match_property.go

```diff
@@ -258,6 +258,17 @@ func (p *Projection) matchProperty(_ *requiredProperty, childPlanInfo ...*physic
 }
 
 // matchProperty implements PhysicalPlan matchProperty interface.
+func (p *Analyze) matchProperty(_ *requiredProperty, childPlanInfo ...*physicalPlanInfo) *physicalPlanInfo {
```

Comment:
It's not reasonable to implement `matchProperty` which means trying to match the required property. But Analyze's property is always null. It's ok to put this in `convert` logic